### DISCUSSION TITLE
🐛 Next episode doesn’t start automatically if maximise the video during the next-up screen

### DISCUSSION THREAD

**Discussion by syfq91** [DonutWare/Fladder#157] [Ideas]
### Description

If I maximise the video during the next-up screen, it will just stop at the end of the current episode. It won’t automatically play the next episode. 

But if I let the current episode to finish with the next-up screen visible, it will automatically play the next episode. 

### Reproduction steps

```bash
1. Play video 
2. Maximise video during next-up screen
3. Let video play to the end 
4. Next episode doesn’t start
```

### Screenshots

![Image](https://github.com/user-attachments/assets/82781e96-53a0-4d15-9465-6cff6718ee25)

### Logs

```bash

```

### Platform

iOS

### App Version

V0.4.4 build 78

**Comment by PartyDonut** [comment]
@syfq91 

This is the intended behaviour, I wasn't sure on what to name or show that button as.

I'm open to ideas/discussion on this however because I agree it's not really obvious that is what is going on. 
So for now I have converted the issue to this discussion.

**Comment by syfq91** [comment]
I think the button icon is fine. 

If I maximise the video, can you make it to show the next-up screen again at the end of the video? So I can continue to play next episode 

And if you show it again, maybe change the pause button to a play button. 
![IMG_0283](https://github.com/user-attachments/assets/2711669b-992e-4eed-bb47-4d8ddaa8ef23)

**Comment by syfq91** [comment]
Some more suggestion. 

Shouldn’t it be better if the countdown button on the top left is changed to a play button? I guess users can click it to immediately play the next episode. 

Right now it has 2 pause buttons. If you change the next-up pause button to a play button, maybe you can remove the next button on the bottom left. 

![IMG_0283](https://github.com/user-attachments/assets/4f5faaac-925d-4846-9a75-30e4cf0365b8)
